docs/www/guide-wasm-elastic.md

```diff
@@ -1,261 +1,285 @@
+    coproc_enable: true
+    ```
 
 ## Create the Wasm script
-In this example we use a folder for save all Wasm scripts, `/wasm_example`
-
-To create a Wasm script we will use `rpk wasm generate wasm-elastic`. It 
-will create a template giving us a simple build system and basic example script.
-
-```bash
-$ rpk wasm generate wasm-elastic
-npm created project in /wasm_example/wasm
-``` 
-That command creates a folder with an example Wasm
-script in `/wasm_example/wasm-elastic`. we move to `wasm-elastic` folder.
-
-```bash
-$ cd wasm_example/wasm-elastic
-
-$ tree . 
-  .
-  ├── package.json
-  ├── src
-  │   └── wasm.js
-  ├── test
-  │   └── wasm.test.js
-  └── webpack.js
-```
-Now we just to go `src/wasm.js`, and rename to `src/elastic.js`
-
-```bash
-$ mv src/wasm.js src/elastic.js
-```
-
->Note: the file name is very important, since it will be used to uniquely 
-identify the function when we upload it to the cluster. 
-
-```bash
-$ tree src
- src
-  └── elastic.js
-```
-
- Be sure to add Elasticsearch to your install dependencies:
-
-```bash
-$ npm install @elastic/elasticsearch
-```
-
-Open `src/elastic.js` in your editor of choice and paste the 
-following javascript code. We'll walk through each of the 
-sections to better understand what they do and how they fit 
-together.
-
-```js
-const {
-  SimpleTransform,
-  PolicyError,
-} = require("@vectorizedio/wasm-api");
-
-const {Client} = require('@elastic/elasticsearch')
-
-const client = new Client({ node: 'http://localhost:9200' })
-
-const transform = new SimpleTransform();
-/* Topics that fire the transform function */
-transform.subscribe(["origin"]);
-/* The strategy the transform engine will use when handling errors */
-transform.errorHandler(PolicyError.Deregister);
-/* function for validating record has the correct header key and value  */
-const containHeader = (key, value, record) =>
-  record.headers.some( header =>
-    header.headerKey.equals(Buffer.from(key)) &&
-    header.value.equals(Buffer.from(value))
-  )
-
-/* Transform function */
-transform.processRecord((recordBatch) => {
-  console.log("Applying Wasm process function")
-  return Promise.all(
-    recordBatch.records.map((record) => {
-      if (containHeader("logService", "elastic", record)) {
-        console.log("Index record value to Elasticsearch")
-        return client.index({
-          index: "result_wasm",
-          type: "redpanda_wasm",
-          body: {
-            /*
-             index the record value into Elasticsearch
-            */
-            record_message: record.value.toString()
+
+To create a Wasm script we'll use the `rpk wasm generate` command.
+This command creates a template that gives us a simple build system and basic example script.
+
+Let's put all of our work for this tutorial in `/wasm_example` so it's easy to clean up.
+
+1. Generate the Wasm template:
+
+    ```bash
+    rpk wasm generate /wasm_example/wasm-elastic
```

Comment:
let's not use `/` since it will generate on the root. filesystem

i would just do `rpk wasm generate hello-elastic`